src/main.cpp

```diff
@@ -2264,6 +2267,7 @@ static bool ActivateBestChainStep(CValidationState &state, CBlockIndex *pindexMo
  * that is already loaded (to avoid loading it again from disk).
  */
 bool ActivateBestChain(CValidationState &state, const CBlock *pblock) {
+    const CChainParams& chainparams = Params();
     CBlockIndex *pindexNewTip = NULL;
     CBlockIndex *pindexMostWork = NULL;
     const CChainParams& chainParams = Params();
```

Comment:
I'm going to maintain chainparams instead of chainParams for uniformity across the codebase.
When @theuni asked me #XXXX not to use "params" for both CChainParams and Consensus::Params I used "consensusParams" and "chainparams", but he meant "chainParams" (btw pow.cpp still uses "params" #5812, no big deal).
Now I'm afraid is too late for chainParams, even if he used that in one of his improvements to the checkpoints code #YYYY.
I can dig all the PR numbers if anyone is interested, but I don't have them at hand.
Sorry for the misunderstanding between theuni and me, mostly my fault since I was the listener.
I don't think theuni is married to chainParams, I'm sure he's fine with chainparams, he seemed mostly concerned about using the same variable name for 2 different types, I was just removing "Params()." all around.